### ISSUE TITLE
oram: various versions of pianoteq do not work 

### ISSUE THREAD

**Issue by pft** [zynthian/zynthian-issue-tracking#1194]
**Describe the bug**
On a fresh installation of oram, installed with 2024-06-26-zynthianos-bookworm-aarch64-oram-2406, pianoteq gives no sound. When I inspected the program via vnc, I got this screen:, stating the included version is outdated: 

![too old pianoteq|634x500](https://discourse.zynthian.org/uploads/default/original/2X/7/78508fd52a3738e346a7a7352ab5f70bae7fa7bb.png)

Then I updated it with the latest pianoteq stage, which is pianoteq_stage_linux_trial_v831.7z, which did work.

So for the next version, if not planned already, best include version 8 of pianoteq.

But I've purchased pianoteq 6, so wanted to upload that. However, that does not install right at all, as you can see below, the pianoteq symlink points to a non-existing file. 

```
(venv) root@zynthian:/zynthian/zynthian-sw/pianoteq6# ls -alh
total 64K
drwx------  3 root root 4.0K Sep  7 09:54 .
drwxr-xr-x 11 root root 4.0K Sep  7 09:50 ..
drwxr-xr-x  2 root root 4.0K Jul 31  2020 Documentation
-rw-r--r--  1 root root  42K Sep 19  2017 Licence.rtf
-rw-r--r--  1 root root 6.5K Sep 19  2017 README_LINUX.txt
lrwxrwxrwx  1 root root   48 Sep  7 09:54 pianoteq -> '/zynthian/zynthian-sw/pianoteq6/Pianoteq 6 STAGE'
```

It seems the structure of the installation files sometimes changes between versions, and that not the right files are extracted from the 7z file. The files pertaining to the arm version in the 7z file are listed below:

```
   266348 2020-07-31 14:40:30 Pianoteq 6/arm/extra/lame
 39485408 2020-07-31 14:40:30 Pianoteq 6/arm/Pianoteq 6
       28 2020-07-31 14:40:34 Pianoteq 6/arm/Pianoteq 6.so
 37374192 2020-07-31 14:40:34 Pianoteq 6/arm/Pianoteq 6.lv2/Pianoteq_6.so
     1647 2020-07-31 14:40:34 Pianoteq 6/arm/Pianoteq 6.lv2/manifest.ttl
    50172 2020-07-31 14:40:34 Pianoteq 6/arm/Pianoteq 6.lv2/Pianoteq_6.ttl
    50668 2020-07-31 14:40:34 Pianoteq 6/arm/Pianoteq 6.lv2/Pianoteq_6_(5_channels).ttl
        0 2020-07-31 14:40:34 Pianoteq 6/arm/Pianoteq 6.lv2
        0 2020-07-31 14:40:30 Pianoteq 6/arm/extra
        0 2020-07-31 14:40:34 Pianoteq 6/arm
```

**Expected behavior**
Working pianoteq. Note by the way that version 6 did work on earlier versions of zynthian,

**Comment by riban-bw** [comment]
The demo version of Pianoteq bundled with zynthian-2409 (Oram) has been updated to V8.

Unfortunately it will not be possible to run Pianoteq V6 on zynthian-2409 (Oram). This is because Modartt only produced a 32-bit ARM version of V6 and Oram is 64-bit only. Modartt have actually removed the ARM version of V6 from their download page which means you can no longer obtain it and must rely on previously downloaded copy. Modartt offer an upgrade option for €29 which is the only option you have if you want to use Pianoteq on Oram. Your alternative is to leave your zynthian running zynthian-2401 which is the last stable 32-bit version of zynthian.

It pains us to make this decision but there is no alternative. Sorry!

**Comment by pft** [comment]
Hi @riban-bw you just beat me to the same conclusion. I saw the change in commit 524079109b4c25b7e4ed1642b9f9fc0db2aa5491 of zynthian-sys, which explained the faulty symlink in the unpacked 7z. 

By the way, from the user area I am still able to download the ARM version. 

Thought about trying multiarch, but it isn't worth my time. I'll just shell out some money and fly with version 8.

Oh and I saw that the default pianoteq demo version has also been updated, so that's solved as well.

Thanks!
